layouts/projects/projects.html

```diff
@@ -0,0 +1,14 @@
+{{ partial "header.html" . }}
```

Comment:
If you have time to work on this, I'd like for us to try to use default layout here instead of adding a new one. I think it should be possible. Otherwise we can do it in a followup pull request.

Reply:
I don't understand how to specify that a particular file should use _default/single.html as its template

Reply:
I don't think _default/section.html template is being used anywhere. Can I change the section.html file to look like my projects.html file? That works, I tried, and the template is very basic - so I don't think it should hurt. The current section.html template is pretty redundant.

Reply:
This way, I'll be able to use section.html for the other PR about documentations menu as well.

Reply:
Good call!